include/z64scene.h

```diff
@@ -258,17 +258,16 @@ typedef struct {
     u32 dListEnd;
 } MeshHeader2;
 
-
 typedef struct {
-    u8 ambientClrR, ambientClrG, ambientClrB;
-    u8 diffuseClrA_R, diffuseClrA_G, diffuseClrA_B;
-    u8 diffuseDirA_X, diffuseDirA_Y, diffuseDirA_Z;
-    u8 diffuseClrB_R, diffuseClrB_G, diffuseClrB_B;
-    u8 diffuseDirB_X, diffuseDirB_Y, diffuseDirB_Z;
-    u8 fogClrR, fogClrG, fogClrB;
-    u16 unk;
-    u16 drawDistance;
-} LightSettings;
+    /* 0x00 */ Color_RGB8 ambientColor;
+    /* 0x03 */ s8 diffuseDir1[3];
```

Comment:
these colors need to be u8 arrays to match in z_lights, and I believe in kankyo as well. So I dont think using `Color_RGB8 ` here is a good idea

Reply:
not to mention the data structure for the lights in gbi is using u8 arrays as well

Reply:
Where at specifically? 🤔

Reply:
what do you mean, sorry

Reply:
should we also move this struct into z64lights.h? would make more sense there, could also call it `EnvironmentLightSettings` but that would break ZAPD.

Reply:
I meant where does it not match with the struct, but either way I just changed it

Reply:
I think it will make more sense in kankyo, since thats where the settings are most relevant. I already have the struct there in my branch. So this struct is kind of temp tbh

Reply:
alright, well it can stay here for the time being then.